src/helpers.js

```diff
@@ -0,0 +1,10 @@
+
+async function awaitExecution(child) {
```

Comment:
maybe an optional timeout can be useful here to ensure this eventually finishes executing?

Reply:
probably difficult to come up with a default timeout, maybe you can also check if there has been any stdout or stderr messages and if there are none for 10minutes or more, have it fail.

Reply:
@christian-bromann : I added a timeout of 2 minutes on this without no console output.
Since saucectl is expected (during tests) to print `.` every second or so, this should be enough.

2 minutes seems also enough for uploading the test payload knowing Github connectivity.